velox/common/encode/Base64.h

```diff
@@ -15,33 +15,42 @@
  */
 #pragma once
 
+#include <array>
```

Comment:
Can we move this to the cpp file?

Reply:
https://github.com/facebookincubator/velox/blob/ccc856eafdf07ac7e77580ec8fb4b764d04763a4/velox/common/encode/Base64.h#L35

The above code uses array, That's the reason I added this header